src/NzbDrone.Core/Datastore/Migration/170_add_MaximumSingleEpisodeAge_to_indexers.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(170)]
```

Comment:
Please change to 172, 170 and 171 are already spoken for in work that is in progress.